src/styles/index.scss

```diff
@@ -0,0 +1,14 @@
+@import "./utils/reset";
+@import "./utils/variables";
+@import "./blocks/page.scss";
+@import "./blocks/header.scss";
+@import "./blocks/logo-link.scss";
+@import "./blocks/nav.scss";
+@import "./blocks/is-active.scss";
+@import "./blocks/content.scss";
+@import "./blocks/card.scss";
+@import "./blocks/stars.scss";
+@import "./blocks/reviews.scss";
+@import "./blocks/price-ru.scss";
+@import "./blocks/price.scss";
+@import "./blocks/link-buy.scss";
```

Comment:
so many files and BEM blocks, it shouldn't be like this. Inside the card should be only `stars` as a BEM block, all other should be elements